cmd/config/docs/api-conventions/functions-spec.md

```diff
@@ -0,0 +1,160 @@
+# Configuration Functions Specification
+
+This document specifies a standard for client-side functions that operate on Kubernetes declarative configurations. This standard enables creating small, interoperable, and language-independent executable programs packaged as containers that can be chained together as part of a configuration management pipeline. The end result of such a pipeline are fully rendered configurations that can then be applied to a control plane (e.g. Using ‘kubectl apply’ for Kubernetes control plane). As such, although this document references Kubernetes Resource Model and API conventions, it is completely decoupled from Kuberentes API machinery and does not depend on any in-cluster components.
+
+This document references terms described in [Kubernetes API Conventions][1].
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119][2].
+
+## Applications
```

Comment:
Use Cases?  Motivation?

Applications is confusing as a term -- e.g. App as in workload or App as in use case

Reply:
Renamed to Use Cases.